src/app/types/finalityProviders.ts

```diff
@@ -15,3 +17,19 @@ export interface Description {
   securityContact: string;
   details: string;
 }
+
+export interface FinalityProvidersProps {
+  finalityProviders: FinalityProvider[] | undefined;
+  selectedFinalityProvider: FinalityProvider | undefined;
+  onFinalityProviderChange: (btcPkHex: string) => void;
+  queryMeta: QueryMeta;
+}
+
+export type SortField = "moniker" | "btcPk" | "stakeSat" | "commission";
```

Comment:
Okay with `asc` `desc` but can we extract and reuse constants "moniker", "btcPk", "stakeSat", "comission". We use it as strings in `const criteria: Criteria` `src/app/components/Modals/FinalityProviderMobileSortModal.tsx`